app/components/search-result.js

```diff
@@ -27,6 +27,14 @@ export default Ember.Component.extend(Analytics, {
         return result.description.slice();
     }),
 
+    contributors: Ember.computed('result', function() {
+        let result = this.get('result');
+        if (result.contributors) {
+            return result.contributors.reverse();
```

Comment:
So did switching a and b on https://github.com/CenterForOpenScience/ember-preprints/blob/develop/app/controllers/discover.js#L179 not work?